### ISSUE TITLE
SCG method in GPyTorch

### ISSUE THREAD

**Issue by takafusui** [cornellius-gp/gpytorch#1627]
Hi,

Are there any implementations where the scaled conjecture gradient (: SCG) method is used to optimize the model's hyperparameters, instead of LBFGS or the stochastic gradient descent?

**Comment by jacobrgardner** [comment]
No, primarily because there weren't any good implementations of nonlinear conjugate gradients last time we looked.

Is there a reason you need this specifically over something like LBFGS?

**Comment by takafusui** [comment]
Thank you @jacobrgardner for your answer. Not really. I usually use either adam or PyTorch-LBFGS to train GP.
I want to try SCG and compare it with the other two optimizers.

**Comment by gpleiss** [comment]
@takafusui we're open to a PR, if you'd like to implement it! 😉
